src/TipBox.vue

```diff
@@ -54,50 +57,57 @@
       heading: {
         type: String,
         default: null,
-      }, 
+      },
       light: {
         type: Boolean,
         default: false,
-      }
+      },
+      seamless: {
+        type: Boolean,
+        default: false,
+      },
     },
     computed: {
       isDefault() {
-        return this.type === 'none'
+        return this.type === 'none';
+      },
+      isSeamless() {
+        return toBoolean(this.seamless);
       },
       boxStyle() {
         switch (this.type) {
+        case 'warning':
+          return 'alert-warning'
+        case 'info':
+        case 'definition':
+          return 'alert-info'
+        case 'success':
+        case 'tip':
+          return 'alert-success'
+        case 'important':
+        case 'wrong':
+          return 'alert-danger'
+        default:
+          return 'alert-default'
+        }
+      },
+      lightStyle() {
+        if (this.light) {
+          switch (this.type) {
           case 'warning':
-            return 'alert-warning'
+            return 'border-warning text-warning alert-border-left';
```

Comment:
This diff is misleading and undesirable, especially in a commit `Implement seamless style tipbox`.

It's better to avoid unrelated and extensive formatting changes in a feature PR.

Reply:
Sorry that was a commit in progress... I tried to revert things back to master to minimize the diff, now I finished doing that - it's a 40-line PR now.